### PR TITLE
feat: add sorted set remove elements

### PR DESCRIPTION
**Pull request by malandis**
Adds the sorted set remove elements api, documentation, response
types, and integration tests.

### CODE REVIEW

File: src/Cache/CacheClient.php [hunk 1]
```diff
@@ -2010,7 +2011,53 @@ public function sortedSetRemoveElement(string $cacheName, string $sortedSetName,
...
+     * Remove elements from a sorted set.
+     *
+     * @param string $cacheName Name of the cache that contains the sorted set.
+     * @param string $sortedSetName The set to remove the elements from.
+     * @param array $values The values to remove.
+     * @return ResponseFuture<SortedSetRemoveElementsResponse> A waitable future which
+     * will provide the result of the sorted set remove elements operation upon a blocking call to
+     * wait.
+     * <code>$response = $responseFuture->wait();</code><br />
+     * The response represents the result of the sorted set remove element operation.
```
Comment (nand4011) [inline src/Cache/CacheClient.php:2024]:
```suggestion
     * The response represents the result of the sorted set remove elements operation.
```

File: src/Cache/CacheClient.php [hunk 2]
```diff
@@ -2010,7 +2011,53 @@ public function sortedSetRemoveElement(string $cacheName, string $sortedSetName,
...
+        return $this->getNextDataClient()->sortedSetRemoveElements($cacheName, $sortedSetName, $values);
+    }
+
+    /**
+     * Remove elements from a sorted set.
+     *
+     * @param string $cacheName Name of the cache that contains the sorted set.
+     * @param string $sortedSetName The set to remove the elements from.
+     * @param array $values The values to remove.
+     * @return SortedSetRemoveElementsResponse Represents the result of the sorted set remove element operation.
```
Comment (nand4011) [inline src/Cache/CacheClient.php:2049]:
```suggestion
     * @return SortedSetRemoveElementsResponse Represents the result of the sorted set remove elements operation.
```

**Review by nand4011 — APPROVED** [review]
Tiny comment nits that can wait for the next pr.

File: src/Cache/CacheClient.php (hunk 2 above)
Comment (malandis) [reply src/Cache/CacheClient.php:2049]: Thanks. Will correct in a follow up.

**Pull request merged**